### ISSUE TITLE
Indentation changed after `puni-kill-line`

### ISSUE THREAD

**Issue by twlz0ne** [AmaiKinono/puni#6]
Following are the test code:

``` elisp
(cl-labels ((test-kill
             (kill-function)
             (with-temp-buffer
               (insert (concat "if cond1:\n"
                               "    True\n\n"))
               (insert (concat "if cond2:\n"
                               "    False\n"))
               (python-mode)
               (goto-char (point-min))
               (re-search-forward "if" nil t 2) ;; move to the 2nd `if'
               (let ((kill-whole-line nil))
                 (funcall kill-function)
                 (insert "|")
                 (substring-no-properties (buffer-string))))))
  (dolist (kill-function '(puni-kill-line awesome-pair-kill paredit-kill))
    (princ (format "%s:\n" kill-function))
    (print (test-kill kill-function))))
;; =>
;; puni-kill-line:
;; 
;; "if cond1:
;;     True
;; 
;;     if|
;;     False
;; "
;; awesome-pair-kill:
;; 
;; "if cond1:
;;     True
;; 
;; if|
;; "
;; paredit-kill:
;; 
;; "if cond1:
;;     True
;; 
;; if|
;; "
```

- macOS 10.12.6
- Emacs 27.2 & 28.0.50-20210819 (rev: fba64e1697174369b87e3de0c189a0fb0963c49c)
- puni-20210821.10 (rev: 29ae364e9028460accd32b54d80d33a6b411602b)

**Comment by AmaiKinono** [comment]
Should be fixed. Please test.

```elisp
(with-temp-buffer
  (insert (concat "if cond1:\n"
                  "    True\n\n"))
  (insert (concat "if cond2:\n"
                  "    False\n"))
  (python-mode)
  (goto-char (point-min))
  (re-search-forward "if" nil t 2) ;; move to the 2nd `if'
  (let ((kill-whole-line nil))
    (puni-kill-line)
    (insert "|")
    (substring-no-properties (buffer-string))))
;; =>
;; "if cond1:
;;     True

;; if|
;;     False
;; "

```

**Comment by twlz0ne** [comment]
Confirmed fixed with da6f1ff746b48e0ae58501bd355b86f942c77ce7

**Comment by AmaiKinono** [comment]
Great! I'll close the issue then.
